### PR TITLE
refactor: remove deprecated function

### LINKED ISSUE #62: [refactor]: Remove ShiftPath and replace with getURLPart
For simplicity the `ShiftPath` should be removed and all its usage should be replace with the `getURLPart`.

`ShiftPath` is takiing the request path and return the last value of the URL and return a modified shrinked version:

```go
// URL: /a/b/c
r.URL.Path, c := ShiftPath(r.URL.Path) // c = "c"
r.URL.Path, b := ShiftPath(r.URL.Path) // b = "b"
```

The simpler `getURLPart` function gets an URL value by index:

```go
// URL: /a/b/c
c := getURLPart(r.URL.Path, 3) // notice that indexes starts at 1, not 0. a is at idx 1, b at 2 and c at 3
```

I'd start this refactor by removing the `ShiftPath` function from `url.go`. In fact, I'd delete the entire `url.go` file and `go build` to fix compile error.

Before replacing a call to `ShiftPath` we need to make sure we know the proper index we want to use with the `getURLPart`.

For this, reverting to the URL in `server.go` for this handler would be how I'd ensure.

### PR DESCRIPTION
**Pull request by VladPetriv**
Remove deprecated `ShiftPath`  function and replace it with `getURLPart`.
Resolve #62

### CODE REVIEW

File: membership.go [hunk 1]
```diff
@@ -197,10 +197,7 @@ func (m *membership) sudoGetTokenFromAccountID(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	id := ""
-
-	_, r.URL.Path = ShiftPath(r.URL.Path)
-	id, r.URL.Path = ShiftPath(r.URL.Path)
+	id := getURLPart(r.URL.Path, 3)
```
Comment (dstpierre) [inline membership.go:200]: the index for this should be 2 if I'm not mistaken, the URL for this is as follow: `/sudogettoken/:accountId`

**Review by dstpierre — COMMENTED** [review]
Hey @VladPetriv can you double check the following, I think the index of the `getURLPart` is erroneous.

File: membership.go (hunk 1 above)
Comment (VladPetriv) [reply membership.go:200]: Sorry for long answer i'll test it now

Comment (VladPetriv) [reply membership.go:200]: Yep it was really mistake

**Pull request merged**